src/Components/Endpoints/src/Builder/RazorComponentEndpointConventionBuilder.cs

```diff
@@ -10,10 +11,7 @@ namespace Microsoft.AspNetCore.Builder;
 /// <summary>
 /// Builds conventions that will be used for customization of <see cref="EndpointBuilder"/> instances.
 /// </summary>
-
-// TODO: This will have APIs to add and remove entire assemblies from the list of considered endpoints
-// as well as adding/removing individual pages as endpoints.
-public class RazorComponentEndpointConventionBuilder : IEndpointConventionBuilder
+public sealed class RazorComponentEndpointConventionBuilder : IEndpointConventionBuilder
```

Comment:
```suggestion
public sealed class RazorComponentsEndpointConventionBuilder : IEndpointConventionBuilder
```

This is one of the things we made plural in https://github.com/dotnet/aspnetcore/issues/49754#issuecomment-1660809201.